python/tool/sphinx/source/conf.py

```diff
@@ -0,0 +1,25 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'typedb-driver'
```

Comment:
We can discuss where these files should live if they are not generic (eg. reusable) and specific to our generation. I'm not sure putting them with the 'generic' tooling is correct yet

Reply:
Moved to the driver root.